sunbeam-python/sunbeam/core/steps.py

```diff
@@ -69,6 +67,7 @@ def __init__(
         config: str,
         application: str,
         model: str,
+        roles: list[Role] | list[list[Role]] = [],
```

Comment:
```suggestion
        roles: list[Role] | list[list[Role]] | None = None,
```

Can we not have mutable defaults?